pkg/transforms/policyreport.go

```diff
@@ -48,12 +48,25 @@ func PolicyReportResourceBuilder(pr *PolicyReport) *PolicyReportResource {
 	node.Properties["apiversion"] = gvk.Version
 	node.Properties["apigroup"] = gvk.Group
 
+	// Overwrite the name to be the namespace - which is same as the cluster the policy affects
+	node.Properties["name"] = pr.Namespace
+	// Total number of policies in the report
+	node.Properties["numInsightPolicies"] = len(pr.Results)
```

Comment:
How will we use this field? Do we expect to use it for searching?  ie. `numInsightPolicies >2` ?

If it's only to display a total, then we could use length of `insightPolicies` instead.

Reply:
Do you mean using len(policies) instead of len(pr.Results)? @jlpadilla

Reply:
No, I meant that we could get the length **on the client side** instead of saving as a field.  It's just an alternative, it will depend on how we use the data.

The field `numInsightPolicies` will be searchable by users, so I wanted to confirm that is the expected behavior.

Reply:
@zlayne Can you comment on the usage of `numInsightPolicies`?

Reply:
This is a good question.. My original thought was to show the number of rule violations a PolicyReport contained (the `numInsightPolicies`). But I dont think searching by this field really has any value to the user, so maybe we just leave that transform to the search-ui?

Reply:
Since we are creating empty policyReports, if there are no violations, won't this field help in filtering out the empty ones? `numInsightPolicies > 0`

Reply:
ok, @SherinV use case sounds valid, we can use it in the suggested search.